contractcourt/commit_sweep_resolver.go

```diff
@@ -252,18 +252,86 @@ func (c *commitSweepResolver) Resolve(_ bool) (ContractResolver, error) {
 
 		signDesc = c.commitResolution.SelfOutputSignDesc
 	)
+
+	// commitScriptTimeLockTaproot is a function that returns the timelock
+	// of a non-htlc commitment output.
+	//
+	// TODO(proofofkeags): move this
+	// NOTE FOR REVIEWERS: This function does not really belong here. That
+	// said, I don't know where it should go. If you have opinions, please
+	// comment.
+	commitScriptTimeLockTaproot := func(script []byte) fn.Option[int32] {
```

Comment:
If we extract this to an normal function, then we can unit test it more easily.

Reply:
Yeah I actually would like to. Do you have opinions on where it should go?

Reply:
`input` sounds good, there's a `script_utils.go` file where we put different defined scripts.

Reply:
Could just be a private function in this package, since not used elsewhere.

Reply:
Turns out we don't need any of it anymore.